sci-misc/vitables/vitables-2.1-r1.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+MY_P=ViTables-${PV}
+
+DESCRIPTION="A graphical tool for browsing / editing files in both PyTables and HDF5 formats"
+HOMEPAGE="http://vitables.org/"
+SRC_URI="mirror://sourceforge/${PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples"
+
+RDEPEND="
+	dev-python/pytables
+	dev-python/PyQt4[X]"  # FIXME: check if any other useflags are needed
+DEPEND="${RDEPEND}
+	doc? ( dev-python/sphinx )"
+
+S=${WORKDIR}/${MY_P}
+
+PATCHES=( "${FILESDIR}"/${P}-no-docs.patch )
+
+src_compile() {
+	distutils-r1_src_compile
+	# fixme: multiple python (anyone cares?)
+	use doc && PYTHON=python2.7 esetup.py build_sphinx
+}
+
+src_install() {
```

Comment:
use python_install_all here too (don't forget to call `distutils-r1_python_install_all` at the end).